src/pages/LogInWithShortLivedTokenPage.js

```diff
@@ -57,9 +67,14 @@ class LogInWithShortLivedTokenPage extends Component {
         const email = lodashGet(this.props.route.params, 'email', '');
         const shortLivedToken = lodashGet(this.props.route.params, 'shortLivedToken', '');
 
+        // If the account is loading don't send more requests
```

Comment:
This comment is ambiguous and made inspired a few questions...

- which requests we don't want to send?
- how does the early return "stop" those requests from sending?
- why is the account in a `loading` state?

Reply:
Yes I think we should specify here that the account is already trying to authenticate and that we don't want to reattempt authenticating.